WordOfTheDay.Api/Controllers/WordsController.cs

```diff
@@ -60,6 +64,8 @@ public async Task<IActionResult> PostWord(Word word)
 
             await _wordsServices.PostWord(word);
 
+            await _publishEndpoint.Publish(new WordInfo(word.Id, word.Email, word.Text, word.AddTime, word.LocationLongitude, word.LocationLatitude));
```

Comment:
I would put this code to Services level, it is feel more like business logic than handling http

Reply:
done